File: SSP.PureInsuranceRestAPIHandler/SSP.PureInsuranceRestAPIHandler/BaseClasses/BaseClaimProcessPerilRecoveryType.cs
namespace SSP.PureInsuranceRestAPIHandler.BaseClasses
{
    public class BaseClaimProcessPerilRecoveryType
    {
        public string TypeCode { get; set; }
        public decimal Amount { get; set; }
        public string RecoveryPartyTypeCode { get; set; }
        public string RecoveryPartyCode { get; set; }
        public bool IsSalvageRecovery { get; set; }
        public string taxGroupCode { get; set; }
        public bool RecoveryAmountSpecified { get; set; }
        public BaseClaimProcessReceiptDetailsType RecoveryDetails { get; set; }
        public bool isReceiptToDate { get; set; }
        public bool isRecoverToDate { get; set; }
        public decimal RecoveryAmount { get; set; }
    }
}

File: Orion/Components/ImportExport/Interface/SiriusImport/RestContracts/BaseClaimProcessPerilRecoveryType.vb
Public Class BaseClaimProcessPerilRecoveryType
    Public Property TypeCode As String
    Public Property Amount As Decimal
    Public Property RecoveryPartyTypeCode As String
    Public Property RecoveryPartyCode As String
    Public Property RecoveryPartyKey As Integer
    Public Property RecoveryPartyTypeKey As Integer
    Public Property IsSalvageRecovery As Boolean
    Public Property taxGroupCode As String
    Public Property RecoveryAmountSpecified As Boolean
    Public Property RecoveryDetails As BaseClaimProcessReceiptDetailsType
    Public Property isReceiptToDate As Boolean
    Public Property isRecoverToDate As Boolean
    Public Property RecoveryAmount As Decimal
End Class
